locksmith/src/controllers/v2/eventsController.ts

```diff
@@ -43,10 +46,14 @@ export const saveEventDetails: RequestHandler = async (request, response) => {
     })
   }
 
+  // TODO: We should update the metadata on the locks to point to this event
```

Comment:
This will come later. When a lock is added to an event, we should update some of the metadata on the lock to include/point to the event metadata such as date, time, location... etc